tempesta_fw/http_msg.c

```diff
@@ -527,7 +527,6 @@ tfw_http_msg_hdr_xfrm(TfwHttpMsg *hm, char *name, size_t n_len,
 			if (tfw_http_msg_grow_hdr_tbl(hm))
 				return -ENOMEM;
 		orig_hdr = &ht->tbl[hid];
-		BUG_ON(!TFW_STR_EMPTY(orig_hdr));
```

Comment:
Why? Typically we can see empty TfwStr as a header for special headers directly indexed while raw header must not be empty: if we added the header, then it shouldn't be empty. Meantime, if we deleted the header previously, then why do we try to update/delete/add it again?